src/data.js

```diff
@@ -21,9 +21,9 @@ genotet.data.uploadURL;
 
 /**
  * Genes with binding data available in the Genotet system.
- * @type {!Array<string>}
+ * @type {!Object<string>}
  */
-genotet.data.bindingGenes = [];
+genotet.data.bindingGenes = {};
```

Comment:
I think this should be bindingFiles which is an array of the server binding files info.